### PR TITLE
DTW-75 Updated tech docs landing page to give more info on what they …

### PR DESCRIPTION
**Pull request by ImogenCraigmile**
…contain and their purpose

## Why

The landing page is currently very brief and doesn't provide much information on what the tech docs contain and why they are useful for people who want to use them

## What

Added more information to clarify what the tech docs contain and can be used for 

## Technical writer support

The suggested changes have been through two rounds of content crits and been reviewed by tech writers but still open to opinions on if the content is informative enough for the tech doc users

## How to review

- Check the content for grammar and style
- Confirm that the links work
- Confirm that is displays correctly locally

### CODE REVIEW

File: source/index.html.md.erb [hunk 1]
```diff
@@ -1,16 +1,26 @@
...
 # GOV.UK One Login
 
-GOV.UK One Login is the new way for government services to sign in their users.
+GOV.UK One Login is the way for government services to:
 
-GOV.UK One Login will allow users to access any government service using the same username and password.
+* sign in their users
+* prove their users' identity
+
+This technical documentation will help you understand how GOV.UK One Login works. It gives you information on how you:
```
Comment (jankishah-gds) [inline source/index.html.md.erb:15]: this tech doc will help you understand how you can use GOV.UK One Login?

Comment (charleemg) [inline source/index.html.md.erb:15]: "how you" sounds somehow odd to me? How about "...information on how to"?

Comment (charleemg) [reply source/index.html.md.erb:15]: I do wonder if it does need something more specific. "This technical documentation tells you how to use GOV.UK One Login to xxxx [something in summary of what this page is helping people with, if possible?]

**Review by charleemg — APPROVED** [review]

File: source/index.html.md.erb (hunk 1 above)
Comment (elenaschwan) [reply source/index.html.md.erb:15]:
```suggestion
This technical documentation gives you information on how to:
```

Perhaps we can just simplify it and shorten it to this?

**Comment by huwd** [comment]
@elenaschwan / @sabrinaharris / @ImogenCraigmile 

Will leave you to hit merge on this,
is that likely to be soon? Ideally we don't want approved branches going stale against `main` if they're good to go?

**Comment by elenaschwan** [comment]
I think we wanted one fresh pair of eyes on it - so once Sabrina had a chance to review it, we should be able to merge.

File: source/index.html.md.erb [hunk 2]
```diff
@@ -1,16 +1,26 @@
...
+
+* plan the functionality your service needs
+* register your service with GOV.UK One Login
+* integrate with GOV.UK One Login to authenticate users and prove their identity
+* go live with your integration
 
 You can [read further documentation about how GOV.UK One Login works](/how-gov-uk-one-login-works/).
 
+Contact us if you have any questions or feedback on our [#govuk-one-login](https://ukgovernmentdigital.slack.com/archives/C02AQUJ6WTC) Slack channel.
```
Comment (sabrinaharris) [inline source/index.html.md.erb:24]:
```suggestion
Contact us if you have any questions on our [#govuk-one-login Slack channel](https://ukgovernmentdigital.slack.com/archives/C02AQUJ6WTC).
```
It'd be useful to extend this link so it has more context in isolation 👍 

We don't have a good process in place yet to make sure feedback is picked up on Slack, so I'd like to keep it to questions for now until we have that figured out!

**Review by sabrinaharris — COMMENTED** [review]
This looks great! I think there's an outstanding comment on line 15 aside from my suggestion, then it's good to go 🚀

**Review by sabrinaharris — APPROVED** [review]
Great work :)

**Pull request merged**